webboot/webboot.go

```diff
@@ -153,7 +148,16 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	file, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	body, err := ioutil.ReadAll(read)
+	if err != nil {
```

Comment:
get rid of lines 156-163. you don't need them.

Reply:
Sounds good! I realized I do not need this if I can just open the persistent memory device and copy the content over to it.